src/compile.js

```diff
@@ -511,6 +511,58 @@ Compiler.prototype.cyield = function(e) {
     return "$gen.gi$sentvalue"; // will either be none if none sent, or the value from gen.send(value)
 };
 
+Compiler.prototype.cyieldfrom = function (e) {
```

Comment:
In python all this work is done in `compiler_sync_comprehension_generator` I'm slightly worried we're trying to keep the compiler looking like the python2 compiler 😄 

Not something we should fix in this PR just something I wanted to flag.

Reply:
I used the code from ceval.c `Target(YIELD_FROM)` for the general structure.
https://github.com/python/cpython/blob/main/Python/ceval.c#L2563